pkgs/build-support/build-typst-package.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenvNoCC }:
+{
+  nativeBuildInputs ? [ ],
+  buildInputs ? [ ],
+  meta ? { },
+  ...
+}@attrs:
```

```suggestion
{ ... }@attrs:
```

You don't actually use any of them here.